packages/react-dom/src/__tests__/ReactDOMTextarea-test.js

```diff
@@ -20,6 +20,8 @@ describe('ReactDOMTextarea', () => {
   let renderTextarea;
 
   beforeEach(() => {
+    jest.resetModules();
```

Comment:
Is there any particular reason for this addition? I think the only reason we do that is when we have tests that require multiple separate versions of React at the same time.

Reply:
Btw, thanks for the feedback. I really appreciate it 🙂